app/resources/v1/users.js

```diff
@@ -54,12 +49,11 @@ exports.post = async function (req, res) {
       if (credentials.screenName) {
         user = await User.findOne({ where: { id: credentials.screenName } })
       }
-      loginToken = uuidv1()
       if (!user) {
         const newUserData = {
+          _id: uuid(),
```

Comment:
Remove this line - we are no longer using `_id` properties to identify users. A follow up issue to clean up this property from the database model so that it is not inadvertently used again is here. https://github.com/streetmix/streetmix/issues/1905